src/OneBlinkForm.tsx

```diff
@@ -565,14 +566,11 @@ const OneBlinkFormUncontrolled = React.memo(function OneBlinkFormUncontrolled({
   initialSubmission,
   resumeAtElement,
   ...props
-}: OneBlinkFormBaseProps & {
```

Comment:
Have the base props been updated? is that what's going on here?

Reply:
yes, yes.